lib/rules/comma-style.js

```diff
@@ -62,6 +62,49 @@ module.exports = {
         }
 
         /**
+         * Modified text based on the style
+         * @param {string} styleType Style type
+         * @param {string} text Source code text
+         * @returns {string} modified text
+         * @private
+         */
+        function getReplacedText(styleType, text) {
+            switch (styleType) {
+                case "between":
+                    return `,${text.replace("\n", "")}`;
```

Comment:
Oops, sorry, it was my misread.
I sometimes forget that `String.prototype.replace` replaces only the first matched part.